Processors/6502/Implementation/6502Implementation.hpp

```diff
@@ -327,7 +327,7 @@ template <Personality personality, typename T, bool uses_ready_line> void Proces
 
 							// All flags are set based only on the decimal result.
 							flags_.zero_result = result;
-							flags_.carry = Numeric::carried_out<7>(a_, operand_, result);
+							flags_.carry = Numeric::carried_out<true, 7>(a_, operand_, result);
```

Comment:
This file could be left untouched by this pull request if a signature-compatible `Numeric::carried_out` overload was kept around.

Reply:
I thought it was better to bring `carried_out` into line with `overflow` in that you need to be overt as to whether the operation was an addition or a subtraction.